app/controllers/api/issues_controller.rb

```diff
@@ -19,14 +19,16 @@ def update_issue
 
     def close_issue
       user, repo, number = params[:user], params[:repo], params[:number]
-      @issue = huboard.board(user, repo).issue(number).close
-      render json: @issue
+      @issue = huboard.board(user, repo).issue(number)
+      @issue.close
+      render json: @issue.activities
```

Comment:
Have you considered creating the equivalent event object in memory to send down to the client?

It's possible that @issue.activities could be expensive, ie; several pages long

closing/reopening issues will become slower as the amount of activities goes up